provider/benchmark_test.go

```diff
@@ -23,9 +22,7 @@ import (
 // Runs a benchmark test using the provider BenchmarkConfig
 func RunBenchmark(t *testing.T, bench BenchmarkConfig) {
 	t.Run("benchmark", func(t *testing.T) {
-		if !(BenchmarkFlag.IsSet() || RecordBenchmarkFlag.IsSet()) {
-			t.Skipf("Skip benchmarking because %s", BenchmarkFlag.WhyNotSet())
-		}
+		t.Skip("Skip benchmarking because")
```

Comment:
Am I reading this correctly that this will never run now? C

Reply:
`BenchmarkFlag` and `RecordBenchmarkFlag` were set to a hardcoded value where `IsSet` always returned `false` so these tests never ran (in CI)

Reply:
Old code looked like this:
```go
// providertest doesn't expose a real flags implementation, so just pretend for now
var (
	//("benchmark", "Run benchmarks of provider test programs and compare with recorded values")
	BenchmarkFlag = hardcodedFlag{set: false}
	//("record-benchmark", "Run benchmarks of provider test programs and record results")
	RecordBenchmarkFlag = hardcodedFlag{set: false}
)

type hardcodedFlag struct {
	set bool
}

var _ = flags.Flag(hardcodedFlag{})

func (h hardcodedFlag) IsSet() bool       { return h.set }
func (h hardcodedFlag) WhySet() string    { return "hardcoded" }
func (h hardcodedFlag) WhyNotSet() string { return "hardcoded" }
```